src/action.js

```diff
@@ -161,6 +164,32 @@ const action = async () => {
     prefix + "ActionDependencies"
   );
   console.log("✅ ActionDependencies data sent to Azure Log Analytics");
+
+  // Get central repository data
+  if (getOrgData) {
```

Comment:
Does this get coerced into a boolean by `const getOrgData = core.getInput("get-org-data");`?

I'm wondering if we're going to end up here with every repo with the string `"false"` being truthy.